src/Common/src/TypeSystem/Common/TypeSystemContext.cs

```diff
@@ -661,9 +661,9 @@ public TypeDesc GetSignatureVariable(int index, bool method)
             return _signatureVariables.GetOrCreateValue(combinedIndex);
         }
 
-        protected internal virtual IEnumerable<MethodDesc> GetAllMethods(TypeDesc type)
+        public virtual IEnumerable<MethodDesc> GetAllMethods(TypeDesc type)
```

Comment:
Why do we need this public? This is supposed to be consumed by typesystem provided extension methods.